app.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask, jsonify
+from chatbot import message_to_bot, setup
+
+app = Flask(__name__)
+
+@app.route('/chatbot/<input_user>', methods=['GET'])
+def chat(input_user):
+    clf, learn_response = setup()
+    try:
+        response = message_to_bot(input_user, clf, learn_response)
+    except:
+        return jsonify({'chatbot': "Unable to get response"})
+    
+    return jsonify({'chatbot': response})
```

Comment:
Also, usually, you can pass on a status code along with jsonify. That is, a 200 code for a successful response and a 500 code whenever we catch ALL errors.